### ISSUE TITLE
Force to install in kybe-system

### ISSUE THREAD

**Issue by ranjeetranjan** [Altinity/clickhouse-operator#1116]
Command used to install: **OPERATOR_NAMESPACE=ch ./clickhouse-operator-install.sh** 

```
Setup ClickHouse Operator into 'ch' namespace.
Namespace 'ch' already exists.
Namespace 'ch' exists.
Looks like clickhouse-operator is not installed in 'ch' namespace.
Install operator from manifest /clickhouse/operator/clickhouse-operator-master/deploy/operator/clickhouse-operator-install-bundle.yaml into ch namespace
customresourcedefinition.apiextensions.k8s.io/clickhouseinstallations.clickhouse.altinity.com created
customresourcedefinition.apiextensions.k8s.io/clickhouseinstallationtemplates.clickhouse.altinity.com created
customresourcedefinition.apiextensions.k8s.io/clickhouseoperatorconfigurations.clickhouse.altinity.com created
clusterrole.rbac.authorization.k8s.io/clickhouse-operator-kube-system created
clusterrolebinding.rbac.authorization.k8s.io/clickhouse-operator-kube-system created
the namespace from the provided object "kube-system" does not match the namespace "ch". You must pass '--namespace=kube-system' to perform this operation.
the namespace from the provided object "kube-system" does not match the namespace "ch". You must pass '--namespace=kube-system' to perform this operation.
the namespace from the provided object "kube-system" does not match the namespace "ch". You must pass '--namespace=kube-system' to perform this operation.
the namespace from the provided object "kube-system" does not match the namespace "ch". You must pass '--namespace=kube-system' to perform this operation.
the namespace from the provided object "kube-system" does not match the namespace "ch". You must pass '--namespace=kube-system' to perform this operation.
the namespace from the provided object "kube-system" does not match the namespace "ch". You must pass '--namespace=kube-system' to perform this operation.
the namespace from the provided object "kube-system" does not match the namespace "ch". You must pass '--namespace=kube-system' to perform this operation.
the namespace from the provided object "kube-system" does not match the namespace "ch". You must pass '--namespace=kube-system' to perform this operation.
the namespace from the provided object "kube-system" does not match the namespace "ch". You must pass '--namespace=kube-system' to perform this operation.

```

**Comment by hodgesrm** [comment]
This is using the most current version of the operator on Mac OSX. It seems that the clickhouse-operator-install.sh somehow did not correctly update the namespace when installing. 

I've done this on Linux and not had this problem. I wonder if there's a sed issue or something similar that's causing problems.

**Comment by ranjeetranjan** [comment]
Currently, I am on Mac OS Ventura 13.0

**Comment by Slach** [comment]
when you call deploy/operator/clickhouse-operator-intsall.sh
looks like @sunsingerus was hard-coded manifest.yaml instead of using a template.yaml
behavior was changed Jul 2022
will fix it ASAP, thanks for contributing

**Comment by Slach** [comment]
fixed in 0.21.0
